README.md

```diff
@@ -46,7 +46,7 @@ The video stream is available either via a web stream of via Kinesis.
 
```

Comment:
Please also add some notes regarding the new location of where to edit the hyperparameters and the additional scripts you have added.

Reply:
Should be all right now.